src/web/server/client/src/components/Playback/Playback.tsx

```diff
@@ -0,0 +1,80 @@
+import React, { ComponentState, Component, ChangeEvent } from 'react';
+import ReactPlayer from 'react-player';
+import { Container, FormControl, TextField, Button, Typography } from '@material-ui/core';
+
+interface Props { }
+
+interface State {
+  tmpUrl: string;
+  url: string;
+  vidId: string;
+}
+
+export default class Playback extends Component<Props, State> {
+
+  constructor(props: Props) {
+    super(props);
+    this.state = {
+      tmpUrl: "",
+      url: "",
+      vidId: ""
+    };
+  }
+
+  handleChange = (event: ChangeEvent<HTMLInputElement>): void => {
+    let target: HTMLInputElement = (event.target as HTMLInputElement);
+    let label: string = target.name;
+    let value: string = target.value;
+    this.setState({ [label]: value } as ComponentState);
+  };
+
+  urlSubmit = (): void => {   
+    this.setState({ url: this.state.tmpUrl });
+    this.setState({ tmpUrl: "" });
+  }
+
+  retrieveVideo = (): void => {
+    fetch(`http://localhost:8081/api/getVideo/${this.state.vidId}`)

```

Comment:
You defined a proxy, just call `fetch('/api/getVideo/${this.state.vidId}')`

Reply:
Yes I believe this was branched out before I made the fix.